src/collections/integrations/aerospike-kubernetes-operator/aerospike-kubernetes-operator.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: aerospike-kubernetes-operator 
+subtitle: Collaborative and visual infrastructure as code for aerospike-kubernetes-operator
+integrationIcon: icon/color/aerospike-kubernetes-operator-icon-color.svg
+darkModeIntegrationIcon: icon/white/aerospike-kubernetes-operator-icon-white.svg
+docURL: https://docs.meshery.io/installation/platforms/aks
+category: Platforms
+subcategory: 
+featureList: ["Efficiently deploy and operate your Aerospike clusters.",
+"Automates the configuration, provisioning, scaling, and recovery of Aerospike clusters",
+"Reduces the complexity of manual deployment and lifecycle management."
+]
+workingSlides: [
+ ../_images/meshmap-visualizer.png,
+ ../_images/meshmap-designer.png]
+howItWorks: Collaborative Infrastructure as Code
+howItWorksDetails: Collaboratively manage infrastructure with your coworkers synchronously sharing the same designs.
+published: TRUE
+---
+<p>
+ A Helm chart for Aerospike Kubernetes Operator.
+
+
+</p>
+<p>
+ Collaboratively and visually diagram your cloud native infrastructure with GitOps-style pipeline integration. Design, test, and manage configuration your Kubernetes-based, containerized applications as a visual topology.
+</p>
+<p>
+ Looking for best practice cloud native design and deployment best practices? Choose from thousands of pre-built components in MeshMap. Choose from hundreds of ready-made design patterns by importing templates from Meshery Catalog or use our low code designer, MeshMap, to create and deploy your own cloud native infrastructure designs.
+</p>

```

Comment:
This shouldn't be a part of this PR.

Reply:
How can I fix this ?
and
Is this branch is good https://github.com/sketchomania/layer5/tree/align_internship_programs_cards

Reply:
Try removing the folder, and rebasing the branch.